app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -0,0 +1,39 @@
+.gem-c-intervention {
+  @include govuk-text-colour;
+  @include govuk-responsive-padding(4);
+  @include govuk-responsive-margin(8, "bottom");
+
+  clear: both;
```

Comment:
I can't currently see this rendered so the answer may be obvious, by why is `clear` needed on this element?

Reply:
No, it's not obvious. I copied from `notice` but in hindsight I can't see why it's needed here. Removing.